### ISSUE TITLE
iOS: can't build version 5.6.3, 5.7.0

### ISSUE THREAD

**Issue by ASerga** [Pushwoosh/pushwoosh-react-native-plugin#30]
Hello.
I'm trying to integrate PushWoosh with my iOS app.

I got error till building app in Xcode
![bag](https://user-images.githubusercontent.com/33656379/41051054-bb02d7a4-69bd-11e8-8354-cdeb893f3a3c.png)

Enviroment info
* XCode 9.4
* iOS SDK 11.4
* react-native 0.54.2
* cocoapods 1.4.0

Podfile
```
platform :ios, '9.0'

target 'Preply' do

  pod 'React', :path => '../node_modules/react-native', :subspecs => [
    'Core',
    'CxxBridge', 
    'RCTActionSheet',
    'RCTAnimation',
    'RCTGeolocation',
    'RCTLinkingIOS',
    'RCTSettings',
    'RCTVibration',
    'DevSupport'
  ]
  pod 'yoga', :path => '../node_modules/react-native/ReactCommon/yoga'

  pod 'pushwoosh-react-native-plugin', :path => '../node_modules/pushwoosh-react-native-plugin'


  post_install do |installer|
    installer.pods_project.targets.each do |target|
      if target.name == "React"
        target.remove_from_project
      end
    end
  end

end
```

I tried version 5.6.2. App was builded successfully, but I could not get pushes on mobile device.
I also made step `1.5` and `2` from Native iOS SDK guid https://www.pushwoosh.com/docs/native-ios-sdk
I spent whole day and I gave up to configure PushWoosh.

Maybe I do something wrong?
Thanks a lot!

**Comment by wfhm** [comment]
Hi,

Checking it.

**Comment by wfhm** [comment]
Hey @ASerga 

Got it reproduced. Rolling out the fix will require changing our deploy scripts, however meanwhile you can try using the fork below. Simply download it, rename to pushwoosh-react-native-plugin and  replace the **pushwoosh-react-native-plugin** in your node_modules with it:

https://github.com/wfhm/pushwoosh-react-native-plugin

**Comment by wfhm** [comment]
@ASerga fixed in 5.8.0.
